openapi.yaml

```diff
@@ -9105,6 +9107,305 @@ paths:
                 schema:
                   $ref: "#/components/schemas/WorkspaceWithUsers"
 
+    /v2/logs/exports/{exportId}:
```

Comment:
This spec is used to document user facing APIs. So it should be /v1 instead of /v2. We will internally route it to /v2

Reply:
Oh okay. 
So I am just remove v2, as done in other routes